calcculator-app/src/components/Grid.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import "./styles/Grid.css"
+
+
```

Comment:
Lets try to run our code through a code formatter to make it a little easier to read.

Reply:
We have a lot of whitespace here and could really use a code formatter to make it easier to read.